draft/flutter-driver.md

```diff
@@ -0,0 +1,179 @@
+# Extending Flutter Driver with custom commands
+
+[Flutter Driver](https://flutter.dev/docs/cookbook/testing/integration/introduction) is a library to write end-to-end integration tests for Flutter apps. It's similar to Selenium WebDriver (for web apps), Espresso (for native Android apps) and Earl Grey (for native iOS apps). It works by instrumenting the Flutter app and then driving the application using a suite of Dart tests.
```

Comment:
maybe also link to the other e2e test tools?

Reply:
✅

You can actually use any (Dart) application to interact with Flutter Driver, but it's common to use the Dart test package to write unit tests using the driver. I linked to the Dart test package now.